### PR TITLE
Grid mode on product pages

### PR DESCRIPTION
**Pull request by Williambraecky**
Add more display options for the images on a product page.

TaskId-2833507

### CODE REVIEW

File: addons/website_sale/models/website.py [hunk 1]
```diff
@@ -79,6 +79,26 @@ def _get_product_sort_mapping():
...
 
+    product_page_image_layout = fields.Selection([
+        ('carousel', 'Carousel'),
+        ('grid', 'Grid'),
+        ], default='carousel', required=True,
+    )
+    product_page_image_width = fields.Selection([
+        ('none', 'Hidden'),
+        ('50_pc', '50 %'),
+        ('66_pc', '66 %'), # FIXME: bad name -> 66%
```
Comment (alpa-odoo) [inline addons/website_sale/models/website.py:90]: Why is it a bad name ?

Comment (Williambraecky) [reply addons/website_sale/models/website.py:90]: Nice catch, it's not anymore 🙂 
I forgot to remove the fixme, the value used to be 75%.

**Review by xavierbol — COMMENTED** [review]
Nitpicking 😇 
Apart of that, LGTM, great job. 🙂

**Comment by Williambraecky** [comment]
robodoo r+

**Pull request closed without merging**